src/Utilities.jl

```diff
@@ -187,5 +187,42 @@ function time_to_seconds(s::String)
     error("Uncaught case in computing time from given string.")
 end
 
+"""
+    bilinear_remap!(target_field, source_field)
+
+Interpolate the provided ClimaCore field to the provided ClimaCore space.
+This is not MPI compatible and only works with SingletonCommsContext.
+"""
+function bilinear_remap!(target_field, source_field)
+    target_field .= bilinear_remap(source_field, axes(target_field))
+    return nothing
+end
+
+"""
+    bilinear_remap(source_field, target_space)
+
+Interpolate the provided ClimaCore field to the provided ClimaCore space.
+This is not MPI compatible and only works with SingletonCommsContext.
+This only works for fields of scalars.
+
+This has multiple allocations that could be improved.
+We could also use the Remapper object to improve remapping performance
+when remapping multiple fields between the same spaces.
+"""
+# TODO add test for target_space == source_space (input field is same as output field) - this will replace dummy remap funcitonality
+function bilinear_remap(source_field, target_space)
```

Comment:
add this as a method for `Interfacer.remap` - handles case of CC Field -> CC Field when not identical fields. Can edit current function to return early if `spaces_are_compatible`